Civi/Api4/Action/System/ResetPaths.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Action\System;
+
+/**
+ * Clear CiviCRM caches, and optionally rebuild triggers and reset sessions.
+ *
+ * @method bool getTriggers
+ * @method $this setTriggers(bool $triggers)
+ * @method bool getSession
+ * @method $this setSession(bool $session)
```

Comment:
The docblock looks incorrect. Copy/paste error?

Reply:
Yep :-) will fix!

Reply:
I see that this was copied from the `System::flush` action, and that makes me wonder if this even needs to be a standalone action or if we can just add one more param to that API, along with `triggers` and `session` we could add `paths`.

Reply:
> I see that this was copied from the `System::flush` action, and that makes me wonder if this even needs to be a standalone action or if we can just add one more param to that API, along with `triggers` and `session` we could add `paths`.

That sounds interesting, the reason I made it standalone was following the UI convention where flush and reset paths were separate. Open to reworking this so it's part of flush if that fits better :-)

Reply:
No I think you're right. This is good as-is.